### PR TITLE
Update Dart (1.19.0-dev.0.0) and analyzer (0.27.4-alpha.19).

### PR DESCRIPTION
**Pull request by pq**
Pulls in latest Dart dev SDK and corresponding analyzer package (and updates misc. dependencies).  Should give us a measurable perf bump for flutter analyze and watch (and in the IDE).

/cc @Hixie @abarth

### CODE REVIEW

**Comment by sethladd** [comment]
Does this give us trailing comments? :)

**Comment by pq** [comment]
> Does this give us trailing comments? :)

I **wish** but I just missed the boat with my change which landed early this morning and Bill baked the build late last night. 😞

**Comment by pq** [comment]
Hmmmm, that build error is exciting.  Investigating.

File: packages/flutter_tools/lib/src/commands/test.dart [hunk 1]
```diff
@@ -47,7 +47,6 @@ class TestCommand extends FlutterCommand {
   @override
   bool get requiresProjectRoot => false;
 
-  @override
```
Comment (pq) [inline packages/flutter_tools/lib/src/commands/test.dart]: This override was stale unless the field should actually be `commandValidator`?

@Hixie ?

**Comment by pq** [comment]
Phew.  Clean now.  Ready for @abarth or @Hixie  to take a look! 👍

File: packages/flutter_tools/lib/src/commands/test.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter_tools/lib/src/commands/test.dart]: Yeah, probably commandValidator.

Comment (pq) [inline packages/flutter_tools/lib/src/commands/test.dart]: Fixed.

**Comment by abarth** [comment]
LGTM

**Pull request merged**